cloud/aws/templates/aws_oidc/bin/setup.py

```diff
@@ -30,3 +51,31 @@ def pre_terraform_setup(self):
     def _setup_shared_state_file(self):
         if self.config.use_backend_config():
             backend_setup.setup_backend_config(self.config)
+
+    def requires_post_terraform_setup(self):
+        return True
+
+    def post_terraform_setup(self):
+        for name, doc in SECRETS.items():
+            self._maybe_set_secret_value(
+                f"{self.config.app_prefix}-{name}", doc)
+
+    def _maybe_set_secret_value(self, secret_name: str, documentation: str):
```

Comment:
do we really need types? @clouser2 wdyt?

Reply:
Types are useful for my local development. It allows intellij provide better auto-suggestions. And it helps reading code. Not sure about enabling type-checking on presubmit though. Ideally - yes, but it might require a bunch of gymnastics to make sure all of code passes type-checking.

What is your concern with having types? Inconsistent code style and potentially out-of-date types?

Reply:
I'm indifferent on type hints :). There's definite benefit and having them in a few isolated places likely will increase readability.